apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -232,12 +232,11 @@ const Payment = new GQLListSchema('Payment', {
                     }
                     const receipt = await getById('BillingReceipt', resolvedData['receipt'])
                     const billingContext = await getById('BillingIntegrationOrganizationContext', receipt.context)
+                    const billingIntegration = await getById('BillingIntegration', billingContext.integration)
```

Comment:
What about `deletedAt` filter? Is it necessary here? Cannot remember this detail....